clusters/clusters_api_sdk.go

```diff
@@ -0,0 +1,47 @@
+package clusters
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/compute"
+	"github.com/databricks/terraform-provider-databricks/common"
+)
+
+// Start a terminated Spark cluster given its ID and wait till it's running
+func StartCluster(ctx context.Context, c *common.DatabricksClient, clusterID string) error {
+	_, err := StartClusterAndGetInfo(ctx, c, clusterID)
+	return err
+}
+
+// StartAndGetInfo starts cluster and returns info
+func StartClusterAndGetInfo(ctx context.Context, c *common.DatabricksClient, clusterID string) (*compute.ClusterDetails, error) {
```

Comment:
Can we use WorkspaceClient here directly rather than `common.DatabricksClient`?

Reply:
Updated